pythonquize42.ipynb

```diff
@@ -0,0 +1,99 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "130239a9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "cb2f34c6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def getsalary_record():\n",
+    "    sid=int(input(\"enter id\"))\n",
+    "    sname=input(\"enter name\")\n",
+    "    salary=int(input(\"enter salary\"))\n",
+    "    return[sid,sname,salary]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "88fb9a60",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#create list for a getsalary\n",
+    "record=[\n",
+    "    [1,\"om\",50000],\n",
+    "    [2,\"sai\",40000],\n",
+    "    [3,\"ram\",30000],\n",
+    "    [4,\"sita\",50000],\n",
+    "    [5,\"hanuman\",70000]\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "ead94741",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(\"salary.csv\",\"w\", newline=\"\") as file:\n",
```

Comment:
newline? why we add it

Reply:
we are use of newline because without newline python jupyter add one line in output each row